dspace-server-webapp/src/test/java/org/dspace/app/rest/test/AbstractWorkflowDefinitionIntegrationTest.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.test;
+
+/**
+ * @author Maria Verdonck (Atmire) on 17/12/2019
+ */
+public class AbstractWorkflowDefinitionIntegrationTest extends AbstractControllerIntegrationTest {
```

Comment:
This class seems unnecessary to me, as it's just calling the `setup()` of `AbstractControllerIntegrationTest`.  Is there a reason why we don't just remove this class and use `AbstractControllerIntegrationTest` directly?

Reply:
Removed, wasn't used